README.md

```diff
@@ -98,6 +106,8 @@ export IMAGE_PREFIX=arschles
 make deploy
 ```
 
+The `make deploy` target is sensitive to deltas in version control. In order for you to test local code, you will first need to `git add` and `git commit` the code.
```

Comment:
Same question applies re: whether this should be in a separate PR. However... I have a solution for this (which I recently applied to router's `make deploy` that resolves this adequately such that it doesn't even need to be documented. Let's chat about it later.